GUIDING_PRINCIPLES.md

```diff
@@ -0,0 +1,27 @@
+# Guiding Principles
+
+The Vitess project operates under the guiding principles described below. The PMC is responsible for upholding these principles and keeping this document updated with any changes.
+
+# Mission Statement
+
+*“To be the best performing, most scalable, and available NewSQL storage solution in the Cloud.”*
+
+Vitess can currently run on bare-metal. However, the trend is clear that applications are moving to the cloud. For the foreseeable future, we should continue to support the ability to run on bare-metal, because it will facilitate migration to the cloud.
+
+The Vitess architecture is generic enough to accommodate any other RDBMS in the future. However, Vitess currently runs on MySQL and has many features that provide compatibility with it. Vitess should remain focused on MySQL until all frictions are removed for those who wish to migrate from MySQL to Vitess.
+
+# Vision
+
+The vision of the Vitess project is to serve the software community by gaining popularity and adoption.
```

Comment:
This is super generic :)

Reply:
Intentional for now. We can iterate on this later.